tests/drivers/build_all/sensor/adc.dtsi

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for adc devices
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
+
+/ {
+	adc0: adc {
+		compatible = "zephyr,adc-emul";
+		nchannels = <2>;
+		ref-internal-mv = <3300>;
+		ref-external1-mv = <5000>;
+		#io-channel-cells = <1>;
+		status = "okay";
+	};
+
+	thermistor_R25_10000_B_3974: thermistor-R25-10000-B-3974 {
+		compatible = "zephyr,thermistor-rt-table";
+		/* Format <temp resistance> */
+		tr-table = <(-25) 146676>,
+			   <(-15) 78875>,
+			   <(-5) 44424>,
+			   <(5) 26075>,
+			   <(15) 15881>,
+			   <(25) 10000>,
+			   <(35) 6488>,
+			   <(45) 4326>,
+			   <(55) 2956>,
+			   <(65) 2066>,
+			   <(75) 1474>,
+			   <(85) 1072>,
+			   <(95) 793>,
+			   <(105) 596>,
+			   <(115) 454>,
+			   <(125) 351>;
+	};
+
+	test_adc_b57861s: b57861s@0 {
+		compatible = "zephyr,thermistor";
+		reg = <0x0 0x1000>;
+		io-channels = <&adc0 0>;
+		thermistor = <&thermistor_R25_10000_B_3974>;
```

Comment:
The table etc all look great now. I think that's much easier to read personally -- curious what others think.

As one last question (I swear), I find the name of this property a bit odd. This node "is a" thermistor, so it's weird that it also "has a" thermistor. Maybe `zephyr,rt-table` would be a better choice? Or is this patterned off of an existing  binding you're maintaining compatibility with?

Reply:
Agreed, nice work.

I had the same thought on the property name.

Reply:
Thanks, I appreciate all the help :-)